lib/core/data/user/user_repository_impl.dart

```diff
@@ -27,20 +31,67 @@ class UserRepositoryImpl implements UserRepository {
   }
 
   @override
-  Future<Either<Failure, User>> getUserProfile({ String? userId, String? username }) async {
+  Future<Either<Failure, User>> getUserProfile({String? userId, String? username}) async {
```

Comment:
I defined this before but now please follow GetSomethingInput, 
ex: GetProfileInput with freezed